### ISSUE TITLE
undefined method `frame_line_idx' for an instance of Vernier::Result>

### ISSUE THREAD

**Issue by solnic** [jhawthorn/vernier#84]
Hey hey! I'm trying out Vernier for Sentry so that it could potentially replace StackProf, and my goal is to create an output that has the same structure as StackProf so that it could Just Work with the Sentry backend (at least for a good start).

I looked into the Result class and discovered `each_sample` so I thought I could just use that and then I hit this:

```ruby
result.each_sample { |stack, _| puts stack.frames.map(&:to_s).join("\n") }
#<NoMethodError: undefined method `frame_line_idx' for an instance of Vernier::Result>
```

Seems like a bug?

**Comment by joshuay03** [comment]
Thanks for the report! Fixing in https://github.com/jhawthorn/vernier/pull/88

**Comment by joshuay03** [comment]
Fix has been included in `1.1.2`.
